gnucash/register/register-core/assoccell.h

```diff
@@ -0,0 +1,99 @@
+/********************************************************************\
+ * assoccell.h -- association checkbox cell                         *
+ *                                                                  *
+ * This program is free software; you can redistribute it and/or    *
+ * modify it under the terms of the GNU General Public License as   *
+ * published by the Free Software Foundation; either version 2 of   *
+ * the License, or (at your option) any later version.              *
+ *                                                                  *
+ * This program is distributed in the hope that it will be useful,  *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the    *
+ * GNU General Public License for more details.                     *
+ *                                                                  *
+ * You should have received a copy of the GNU General Public License*
+ * along with this program; if not, contact:                        *
+ *                                                                  *
+ * Free Software Foundation           Voice:  +1-617-542-5942       *
+ * 51 Franklin Street, Fifth Floor    Fax:    +1-617-542-2652       *
+ * Boston, MA  02110-1301,  USA       gnu@gnu.org                   *
+ *                                                                  *
+\********************************************************************/
+
+/** @addtogroup Cell Cell
+ * @{
+ * @file assoccell.h
+ * @struct AssocCell
+ * @brief The AssocCell object implements a cell handler
+ * that will cycle through a series of single-character
+ * values when clicked upon by the mouse and will return a glyph
+ * if font can show it.
+ */
+/* HISTORY:
+ * Copyright (c) 1998 Linas Vepstas
+ * Copyright (c) 2000 Dave Peticolas
+ * Copyright (c) 2001 Derek Atkins
+ * Copyright (c) 2020 Robert Fewell
+ */
+
+#ifndef ASSOC_CELL_H
+#define ASSOC_CELL_H
+
+#include <glib.h>
+
+#include "basiccell.h"
+
+#define GLYPH_PAPERCLIP "\360\237\223\216"
+#define GLYPH_LINK      "\360\237\224\227"
```

Comment:
Please comment these with their unicode code points:
```
#define GLYPH_PAPERCLIP "\360\237\223\216"  // Codepoint  U+1F4CE
#define GLYPH_LINK      "\360\237\224\227"         // Codepoint U+1F517
```
These should be put somewhere that the sheet and report code can get to them too so there's only one definition.

Reply:
Done.